cg/store/filters/status_sample_filters.py

```diff
@@ -132,6 +132,17 @@ def filter_samples_is_not_tumour(samples: Query, **kwargs) -> Query:
     return samples.filter(Sample.is_tumour.is_(False))
 
 
+def filter_samples_by_name_enquiry(samples: Query, name_enquiry: str, **kwargs) -> Query:
```

Comment:
I think "enquiry" is confusing and would like the name to be easier to understand. Something like:
```python
def filter_samples_by_name_pattern(samples: Query, name_pattern: str, **kwargs) -> Query:
    """Return samples matching the name pattern."""
...
```